### DISCUSSION TITLE
Passing more than just the name in the error reference

### DISCUSSION THREAD

**Discussion by nicholasgriffintn** [react-hook-form/react-hook-form#3263] [Q&A] [answered]
Hey,

I'm unsure if this is a thing or not so I figured I'd put it here instead of issues, basically, on the errors object from react-hook-form, I was wondering if it's possible to add more data that the name within the ref.

![image](https://user-images.githubusercontent.com/12116098/97112491-84b4f500-16dc-11eb-8012-4782e171c5dc.png)

For my use case, we have multiple pages on our form, so I'd like to include the page number of the input so I can show validation for the page itself as well.

IE: Page 2 has an invalid input.

Or equally, page 2, 4 and 5 have invalid inputs but 1, 3 and 6 do not

I figure this might be the best place to do that, let me know if there are any ways or other ways of doing this.

**Comment by bluebill1049** [comment, marked as answer]
you should keep those as a global state instead of inside the errors object.

**Reply by nicholasgriffintn** [reply]
Okay, that's what I've done instead I just figured for performance it might be worth looking at. If you think it's better not to then I'll keep along the same lines then. Thanks.
